library/options.inc.php

```diff
@@ -373,7 +373,7 @@ function generate_form_field($frow, $currvalue) {
       " value='" . $formDate . "' $disabled />";
     if (!$disabled) {
        $selector = "#form_{$field_id}";
-       $date_init .= "$('" . $selector . "').datetimepicker({timepicker: false, format:'" . $DateFormat . "'});";
+       $date_init .= "$('" . $selector . "').datetimepicker({timepicker: false,  format:'" . $DateFormat . "'});";
```

Comment:
Actually, this fix may not cause issues here.  I think you are right to leave it out though.  We need to merge your other commit first.